src/docs/product/releases/suspect-commits/index.mdx

```diff
@@ -1,30 +1,59 @@
 ---
-title: Enable Suspect Commits
+title: Suspect Commits
 sidebar_order: 2
 description: "Learn how to associate commits to a release to track suspect commits."
 ---
 
-In your release process, add a step to create a release in Sentry and associate it with commits from your linked repository. We recommend using Sentry’s [Command Line Interface](/product/cli/releases/) or [our API](/product/releases/setup/manual-setup-releases/#using-the-api) to add this step in your release process, especially if you're using [Release Automation](/product/releases/setup/release-automation/).
+In your release process, you can add a step to create a release in Sentry and associate it with commits from your linked repository. When you've done this, you'll be able to:
 
-After this, we can tie together the commits in the release, including:
+- See suspect commits and suggested assignees in issues
+- Resolve issues by commit
+
+If you're using one of our [release automation](/product/releases/setup/release-automation/) options or our [manual setup](/product/releases/setup/manual-setup-releases/) instructions, suspect commits should already be enabled.
```

Comment:
```suggestion
If you're using one of our [release automation](/product/releases/setup/release-automation/) options or our [manual setup](/product/releases/setup/manual-setup-releases/) instructions, suspect commits should already be enabled. However, we've included directions for enabling suspect commits using the [Sentry CLI](#using-the-cli) and [our API](#using-the-api) at the end of this page, just in case you need them. 
```